src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -439,45 +395,72 @@ g.test('sample_count_must_be_valid')
     t.doCreateRenderPipelineTest(isAsync, _success, descriptor);
   });
 
+const kRenderableColorTextureFormatsExtend = [
+  undefined,
+  ...kRenderableColorTextureFormats,
+] as const;
+
 g.test('pipeline_output_targets')
   .desc(
     `Pipeline fragment output types must be compatible with target color state format
   - The scalar type (f32, i32, or u32) must match the sample type of the format.
   - The componentCount of the fragment output (e.g. f32, vec2, vec3, vec4) must not have fewer
     channels than that of the color attachment texture formats. Extra components are allowed and are discarded.
 
+  Otherwise, color state write mask must be 0.
+
   MAINTENANCE_TODO: update this test after the WebGPU SPEC ISSUE 50 "define what 'compatible' means
   for render target formats" is resolved.`
   )
   .params(u =>
     u
       .combine('isAsync', [false, true])
-      .combine('format', kRenderableColorTextureFormats)
+      .combine('writeMask', [0, 0x1, 0x2, 0x4, 0x8, 0xf])
+      .combine('format', kRenderableColorTextureFormatsExtend)
```

Comment:
only used once - maybe just do it inline?
```suggestion
      .combine('format',  [undefined, ...kRenderableColorTextureFormats] as const)
```